src/rules/color-no-invalid-hex/__tests__/index.js

```diff
@@ -0,0 +1,18 @@
+import { ruleTester } from "../../../testUtils"
+import rule, { ruleName, messages } from ".."
+
+const testRule = ruleTester(rule, ruleName)
+
+testRule(null, tr => {
+  tr.ok("a { color: pink; }")
+  tr.ok("a { color: #000; }")
+  tr.ok("a { something: #000, #fff, #ababab; }")
```

Comment:
We should ensure that 4 & 8 digits are working too since they are in latest specs (last digits are for alpha). Maybe we can use a regex if indexOf # > -1 ?